test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/controllers/BusinessListCYAControllerSpec.scala

```diff
@@ -107,6 +109,106 @@ class BusinessListCYAControllerSpec extends ControllerBaseSpec with MockIncomeTa
 
   }
 
+  "submit" should {
+    "return 303, SEE_OTHER" when {
+      "the connector returns successful json and submit with Yes option redirect to Initialse controller" in {
```

Comment:
Could you fix the spelling of Initialise please

Reply:
Done